modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/NotificationTemplateObjectActionExecutorImpl.java

```diff
@@ -93,9 +100,44 @@ public String getKey() {
 		return ObjectActionExecutorConstants.KEY_NOTIFICATION;
 	}
 
+	private Map<String, Object> _getTermValues(
+		ObjectDefinition objectDefinition, Map<String, Object> variables) {
+
+		variables = (Map<String, Object>)variables.get("baseModel");
```

Comment:
@devpedrotavares can we change it to:

```
Map<String, Object> termValues = (Map<String, Object>)variables.get("baseModel");

...

return termValues;
```

since the method name is `_getTermValues`?